co-operative.md

```diff
@@ -0,0 +1,50 @@
+# Hypha Worker Co-operative
+
+## About Us 
+
+Hypha Worker Co-operative Inc. is a worker co-operative incorporated on August 
```

Comment:
```suggestion
_Hypha Worker Co-operative Inc._ is a worker co-operative incorporated on August 
```

Reply:
Why italicize this?

Reply:
Ok with me either way.

Reply:
Gonna mark as resolved, as not necessary, and seems it's not a convention outside my own writing :)